crates/relayer/src/chain/cosmos/ethermint/abi.rs

```diff
@@ -308,15 +307,18 @@ pub fn pack_msg_data(msg: &Any) -> Result<Vec<u8>, Error> {
         )));
     }
 
+    let msg = set_signer(msg, signer, "")?;
```

Comment:
do we need get account_prefix from config?

Reply:
Fixed